maps/~mapsystem/maps.dm

```diff
@@ -165,7 +165,7 @@ var/global/const/MAP_HAS_RANK = 2		//Rank system, also togglable
 	else if(LAZYLEN(lobby_tracks))
 		lobby_track_type = pickweight(lobby_tracks - exclude)
 	else
-		lobby_track_type = pick(subtypesof(/decl/music_track) - exclude)
+		lobby_track_type = pick(decls_repository.get_decls_of_subtype(/decl/music_track) - exclude)
```

Comment:
`get_decl_paths_of_subtype()`, to maintain parity with `subtypesof()` returning a list of typepaths (and also I just don't think it'll work as-is)